sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -138,6 +170,11 @@ internal CryptographyClient(Uri keyId, KeyVaultPipeline pipeline)
         /// </summary>
         public virtual string KeyId => _keyId.ToString();
```

Comment:
`_key` is a string now.

```suggestion
        public virtual string KeyId => _keyId;
```

Reply:
Good catch. The compiler/IntelliSense caught a lot of issues, but since `String.ToString()` still works I totally missed this one.